src/Map.fs

```diff
@@ -12,17 +13,149 @@ type IMap =
 module Map =
     let WidgetKey = Widgets.register<Map> ()
 
+    let MapType =
+        Attributes.defineEnum<MapType> "Map_MapType" (fun _ newValueOpt node ->
+            let map = node.Target :?> Map
+
+            let value =
+                match newValueOpt with
+                | ValueNone -> MapType.Street
+                | ValueSome v -> v
+
+            map.MapType <- value)
+
+    let IsShowingUser = Attributes.defineBindableBool Map.IsShowingUserProperty
+
+    let TrafficEnabled = Attributes.defineBindableBool Map.TrafficEnabledProperty
+
+    let HasScrollEnabled = Attributes.defineBindableBool Map.HasScrollEnabledProperty
+
+    let HasZoomEnabled = Attributes.defineBindableBool Map.HasZoomEnabledProperty
+
+    let MoveToLastRegionOnLayoutChange =
+        Attributes.defineBindableBool Map.MoveToLastRegionOnLayoutChangeProperty
+
+    let RequestedRegion =
+        Attributes.defineSimpleScalarWithEquality<MapSpan> "Map_RequestedRegion" (fun _ newValueOpt node ->
+            let map = node.Target :?> Map
+
+            match newValueOpt with
+            | ValueNone -> ()
+            | ValueSome mapSpan -> map.MoveToRegion(mapSpan))
+
+    let Pins =
+        Attributes.defineListWidgetCollection "Map_Pins" (fun target -> (target :?> Map).Pins)
+
+    let MapElements =
+        Attributes.defineListWidgetCollection "Map_MapElements" (fun target -> (target :?> Map).MapElements)
+
+    let MapClicked =
+        Attributes.defineEvent<MapClickedEventArgs> "Map_MapClicked" (fun target -> (target :?> Map).MapClicked)
+
 [<AutoOpen>]
 module MapBuilders =
     type Fabulous.XamarinForms.View with
 
-        /// Defines a Map widget
-        static member inline Map<'msg>() =
-            WidgetBuilder<'msg, IMap>(Map.WidgetKey, AttributesBundle(StackList.empty (), ValueNone, ValueNone))
+        /// <summary>The Map control is a cross-platform view for displaying and annotating maps</summary>
+        /// <param name ="requestRegion">The region of a map to display when a map is loaded can be set by passing a MapSpan.</param>
+        static member inline Map<'msg>(?requestRegion: MapSpan) =
```

Comment:
I think I would make the region parameter mandatory. I'm not sure it's useful to have a map showing a random location

Reply:
Agreed . Updated :)

Reply:
Updated in my last commit :)